docs/src/appendix/scarb-toml.md

```diff
@@ -78,6 +78,15 @@ url = "http://your.second.rpc.url"
 block_id.number = "123"
 ```
 
+### `[tool.scarb]`
+
+```toml
+[tool.scarb]
+allow-prebuilt-plugins = ["snforge_std"]
+```
+It allow `scarb` to download precompiled dependencies used by `snforge_std`.
```

Comment:
```suggestion
It allows `scarb` to download precompiled dependencies used by `snforge_std` from [the registry](https://scarbs.xyz).
```

Reply:
It would also be good to mention somewhere here that this works only with specific, newer scarb versions